### PR TITLE
Closes #85: Create Folder Structure of Frontend Client

### LINKED ISSUE #85: Create Folder Structure of Frontend Client
**Is your feature request related to a problem? Please describe.**
As a frontend developer I want a clear structure for organising files in the frontend so that I can organise my components

**Describe the solution you'd like**
Folder structure created for organising components and views in frontend

**Describe alternatives you've considered**
Start coding without structure - could cause conflicts between developers

**Additional context**
Blocks:
- #87 
- #80 
- #70 
- #61 
- #60 
- #50 
- #57 
- #52 
- #45

### PR DESCRIPTION
**Pull request by AngelaLH**
Closes #85 

*Initial folder & file structure for frontend

*Folder structure as agreed by frontend team

>src/

>> views/
>>> journal/
>>> dashboard/

>> components/
>>> calendar/
>>> todo/
>>> text/
>>> habit-tracker/

### CODE REVIEW

File: telos-frontend/.vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (RawiriHohepa) [inline telos-frontend/.vscode/launch.json]: Remove this file

**Review by RawiriHohepa — CHANGES_REQUESTED** [review]
package-lock.json should not have been changed

**Review by RawiriHohepa — APPROVED** [review]
Looks good to me

**Comment by ambiyu** [comment]
LGTM

**Review by ambiyu — APPROVED** [review]

**Pull request merged**